alembic/env.py

```diff
@@ -38,8 +38,9 @@ def get_target_metadata():
 
 def get_db_url():
     setup_pythonpath()
-    from backend.util import Config as BackendConfig
-    return BackendConfig.get_config_value(BackendConfig.VALUE_DB_URL)
+    from backend.config import Config as BackendConfig
+    BackendConfig.init()
```

Comment:
Constructor for what? `Config` is a singleton (yep, not so good, but passing it along everywhere is not very much good too), its instance is obtained via `get`, but it should first be initialized via `init`. I’m open to better ideas though.